### PR TITLE
Allow Node parameters and state variables to be assigned NEST probability distributions

### PR DESCRIPTION
**Pull request by clinssen**
See https://github.com/nest/nest-simulator/pull/2482.

### CODE REVIEW

File: pynestml/codegeneration/resources_nest/point_neuron/directives/ReadFromDictionaryToTmp.jinja2 [hunk 1]
```diff
@@ -8,7 +8,11 @@
 {%- if not variable.is_inline_expression %}
 {%- if not variable.is_state() %}
 {{declarations.print_variable_type(variable)}} tmp_{{names.name(variable)}} = {{names.getter(variable)}}();
+{%- if synapse is defined or variable in vector_symbols %}
```
Comment (pnbabu) [inline pynestml/codegeneration/resources_nest/point_neuron/directives/ReadFromDictionaryToTmp.jinja2:11]: Should this condition be `variable not in vector_symbols`?

File: tests/nest_tests/nest_set_with_distribution_test.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                        reason="This test does not support NEST 2")
+    def test_nest_set_with_distribution(self):
+        nest.set_verbosity("M_ALL")
+
+        nest.ResetKernel()
+        nest.Install("nestmlmodule")
+
+        neur = nest.Create("iaf_psc_exp_nestml__with_stdp_nestml", 100)
+        neur.V_abs = nest.random.uniform(0., 1.)    # test setting a state variable
+        neur.V_reset = nest.random.uniform(0., 1.)    # test setting a parameter
```
Comment (pnbabu) [inline tests/nest_tests/nest_set_with_distribution_test.py:72]: Should we also test this for `nest.random.normal`?

File: pynestml/codegeneration/resources_nest/point_neuron/directives/ReadFromDictionaryToTmp.jinja2 (hunk 1 above)
Comment (clinssen) [reply pynestml/codegeneration/resources_nest/point_neuron/directives/ReadFromDictionaryToTmp.jinja2:11]: ``updateValueParam()`` does not work on vectors, hence the condition.

**Review by pnbabu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**